typescript/sdk/src/warp/types.ts

```diff
@@ -59,4 +38,29 @@ export const WarpCoreConfigSchema = z.object({
     .optional(),
 });
 
+// List of constant values for local or interchain fees
+export type FeeConstantConfig = z.infer<typeof FeeConstantConfigSchema>;
+
+// List of chain pairs to blacklist for warp routes
+export type RouteBlacklist = Array<{
+  origin: ChainName;
+  destination: ChainName;
+}>;
+
+// Transaction types for warp core remote transfers
+export enum WarpTxCategory {
+  Approval = 'approval',
+  Transfer = 'transfer',
+}
+
+export type WarpTypedTransaction = TypedTransaction & {
+  category: WarpTxCategory;
+};
+
 export type WarpCoreConfig = z.infer<typeof WarpCoreConfigSchema>;
+
+export interface WarpCoreFeeEstimate {
+  interchainQuote: TokenAmount;
+  localQuote: TokenAmount;
+  localDetails: TransactionFeeEstimate;
+}
```

Comment:
I find this interface slightly confusing
is `localQuote === localDetails.fee`?

Reply:
Not quite, one is a bigint and the other is a TokenAmount, meaning we've resolved the details amount to it's associated token

Reply:
Removing it for a cleaner interface